SQL/user/get_all.sql

```diff
@@ -0,0 +1 @@
+SELECT * FROM users
```

Comment:
Why would you need all users?

Reply:
It's just for the "behind the scene" tab!